flexget/plugins/input/next_trakt_episodes.py

```diff
@@ -75,6 +75,7 @@ def on_task_input(self, task, config):
                     trakt_id = item['show']['ids']['trakt']
                     listed_series[trakt_id] = {
                         'series_name': '%s (%s)' % (item['show']['title'], item['show']['year']),
+                        'series_year': item['show']['year'],
```

Comment:
It has nothing to-do where it is used in code, the user can use the fields via set/if plugins, custom.render(). 
So just to give a example, you want to set the title for a task to 'series_name' + year, assuming you normally omit this from you inputs. Now instead of using 'series_year' the same way you could use 'movie_year', you need specific information what metadata_lookup was used and successfully did the lookup and than separately check the namespace fields like `imdb_year, trakt_series_year, tvmaze_series_year`. My argument is why do i care what meta lookup populated this field as long as its valid, if the user has multiple meta lookup defined, why should he care which of those had a successful lookup and populated the year field, so it can be used?